Assets/Scripts/CSharp7/CSharp7.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class CSharp7 : MonoBehaviour
+{
+    private int _skipCount = 1;
+    private int _takeCount = 2;
+
+    private int _vectorCount = 5;
+    private List<int> xPositions;
+
+    public void Start()
+    {
+        var list = new List<Vector3>(_vectorCount);
+        GetXPositions(list);
+        GetArray();
+    }
+
+    public List<int> GetXPositions(List<Vector3> vectors)
+    {
+        xPositions = vectors
+          .Where(vector => vector.x > _vectorCount)
+          .Select(vector => (int)vector.x)
+          .ToList();
+        return xPositions;
+    }
+
+    private void GetArray()
+    {
+        // Exercise 3.Skip, ToArray
+
+        int[] arraySkip = new int[] { };
+        List<int> arrayConvertSkip = new List<int>(arraySkip);
+
+        arrayConvertSkip = (List<int>)xPositions.ToArray()
```

Comment:
I meant that the skip returns a collection.
And you need to return an array.
Just swap the calls